performance/SwiftUIPerformanceQuickstart/PerformanceExample/Shared/Process.swift

```diff
@@ -16,29 +16,28 @@
 
 import SwiftUI
 import FirebasePerformance
+import FirebaseStorage
 import Vision
 
 class Process: ObservableObject {
   @Published var status: ProcessStatus = .idle
-  @Published var action: ProcessAction?
   @Published var image: UIImage?
+  var uploadSucceeded = false
   var categories: [(category: String, confidence: VNConfidence)]?
   let precision: Float = 0.2
   let site = "https://firebase.google.com/downloads/brand-guidelines/PNG/logo-logomark.png"
 
   #if swift(>=5.5)
-    @MainActor @available(iOS 15, tvOS 15,*) func updateStatusAsync(to newStatus: ProcessStatus) {
+    @MainActor @available(iOS 15, tvOS 15,*)
+    func updateStatusAsync(to newStatus: ProcessStatus, upload: Bool = false) {
```

Comment:
Lets call the method name `updateUploadStatus`. Making it a bit descriptive is easier for us to look back in future. Term "Status" is a bit generic.

Similarly, the parameter `upload` is confusing too. Can we call it 'shouldUpload`?

Reply:
I called it `updateStatus` because it is supposed to update the `ProcessStatus` of the given `Process`, which handles download and classify in addition to upload, which is why I hesitate to single out upload in the name. Perhaps renaming the `upload` parameter to `updateUploadStatus` would make its role clearer?

Reply:
@visumickey do the new changes seem reasonable to you?